index.html

```diff
@@ -68,87 +75,94 @@ <h1>Alex Hamn</h1>
-          />
-          <h3>Multi-Post Stories</h3>
-          <p>
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required. has been the industry's standard dummy text ever
-            since the 1500s, when an unknown printer took a standard dummy text.
-          </p>
-          <ul class="container languages">
-            <li>css</li>
-            <li>html</li>
-            <li>bootstrap</li>
-            <li>Ruby</li>
-          </ul>
-          <button>See Project</button>
+          <img src="https://picsum.photos/684/448?grayscale" alt="placeholder"/>
+          <div class="container project-text">
+            <h3>Multi-Post Stories</h3>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text ever
+              since the 1500s, when an unknown printer took a standard dummy text.
+            </p>
+            <ul class="container languages">
+              <li>css</li>
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <button>See Project</button>
+          </div>
         </article>
         <article class="container project">
-          <img src="https://picsum.photos/327/236?grayscale" alt="placeholder"
-          />
-          <h3>Multi-Post Stories</h3>
-          <p> 
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required. has been the industry's standard dummy text ever
-            since the 1500s, when an unknown printer took a standard dummy text.
-          </p>
-          <ul class="container languages">
-            <li>css</li>
-            <li>html</li>
-            <li>bootstrap</li>
-            <li>Ruby</li>
-          </ul>
-          <button>See Project</button>
+          <img src="https://picsum.photos/684/448?grayscale" alt="placeholder"/>
+          <div class="container project-text">
+            <h3>Multi-Post Stories</h3>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text ever
+              since the 1500s, when an unknown printer took a standard dummy text.
+            </p>
+            <ul class="container languages">
+              <li>css</li>
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <button>See Project</button>
+          </div>
         </article>
       </section>
       <section class="container contact">
         <h2>CONTACT US</h2>
+        <h2>Let's chat!</h2>
         <p> 
           If you have an application you are interested in developing, a feature
           that you need built or a project that needs coding. I’d love to help
```

Comment:
Reduce the width of the form below
### Original design
![right](https://user-images.githubusercontent.com/58376020/125980760-1dc52b8c-9ef5-44f1-b58e-716848037f52.png)

### Your Project
![wrong](https://user-images.githubusercontent.com/58376020/125980787-ae6f686f-c627-40cd-acad-c099bbb2187b.png)

Reply:
Center the button as shown below
### Original design 
![right](https://user-images.githubusercontent.com/58376020/125981021-5d5beb21-120f-48ab-a819-04ca2a9d481b.png)

### Your Project
![wrong](https://user-images.githubusercontent.com/58376020/125981053-372be483-8dd5-4981-ae58-ed4dade713ce.png)